csr.py

```
@@ -24,6 +24,9 @@
 
 import re
 import unicodedata
+import ctypes
```

Ruff (0.11.9)

`ctypes` imported but unused

Remove unused import: `ctypes`

(F401)